resources/js/views/rooms/View.vue

```diff
@@ -381,11 +381,19 @@ export default {
   },
   methods: {
 
+    /**
+     * Get a random refresh interval for polling to prevent
+     * simultaneous request from multiple clients
+     */
+    getRandomRefreshInterval: function () {
+      return Math.abs(env.REFRESH_RATE) + Math.floor(Math.random() * 10);
```

Comment:
This implementation makes the 0-10 seconds of additional refresh rate an unexpected behavior and the 10 is a "magic number" in the code. I'd suggest changing this to subtract for example 15% of the `env.REFRESH_RATE` and then add Math.random() * (10% of `env.REFRESH_RATE`) * 2.